frontend/packages/topology/src/behavior/withCreateConnector.tsx

```diff
@@ -27,9 +27,10 @@ type CreateConnectorWidgetProps = {
     element: Node,
     target: Node | Graph,
     event: DragEvent,
-    choice?: ConnectorChoice,
-  ) => ConnectorChoice[] | void | undefined | null;
+    choice?: ConnectorChoice | React.ReactElement,
```

Comment:
This is incorrect, leave it. We don't callback with a `ReactElement`.
It was a mistake in my pseudo code posted earlier attempting to describe a new solution.

Reply:
fixed